monai/data/meta_tensor.py

```diff
@@ -328,6 +328,51 @@ def as_tensor(self) -> torch.Tensor:
         """
         return self.as_subclass(torch.Tensor)  # type: ignore
 
+    def get_array(self, output_type=np.ndarray, dtype=None, *_args, **_kwargs):
+        """
+        Returns a new array in `ouptut_type`, the array shares the same underlying storage when the output is a
+        numpy array. Changes to self tensor will be reflected in the ndarray and vice versa.
+
+        Args:
+            output_type: output type, see also: :py:func:`monai.utils.convert_data_type`.
+            dtype: dtype of the output array.
+            *_args: currently unused parameters.
+            **_kwargs: currently unused parameters.
```

Comment:
I remember `*` and `**` caused some issue in the web page, other places use `_args`, `_kwargs` directly:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/transforms/croppad/array.py#L1271

Thanks.